source/pug/views/about.pug

```diff
@@ -22,20 +21,20 @@ block content
       .col-md-6.mb-5
         h3.h3-black People
         p.mb-3 Network members are digital citizens of all types: developers, teachers, activists and makers. We span the globe, but we share a singular goal: a healthy Internet.
-        a.cta-link(href=`#TODO`) Meet the network
+        a.cta-link(href=`${env.virtualRoot}people`) Meet the network
```

Comment:
It's so that you can put the site in a subdirectory when you deploy it (eg: for GitHub pages deploys).